contracts/ColonyFunding.sol

```diff
@@ -141,6 +142,89 @@ contract ColonyFunding is ColonyStorage, DSMath {
     return nonRewardPotsTotal[_token];
   }
 
+  function getGlobalRewardPayoutCount() public view returns (uint256) {
+    return globalRewardPayoutCount;
+  }
+
+  function getUserRewardPayoutCount(address _user) public view returns (uint256) {
+    return userRewardPayoutCount[_user];
+  }
+
+  function startNextRewardPayout(address _token) public auth returns(uint256) {
+    require(!activeRewardPayouts[_token]);
+
+    activeRewardPayouts[_token] = true;
+    globalRewardPayoutCount += 1;
+
+    //TODO: Lock everyones tokens
+
+    uint256 amount = pots[0].balance[_token];
+    pots[0].balance[_token] = 0;
+
+    rewardPayoutCycles[globalRewardPayoutCount] = RewardPayoutCycle(
+      IColonyNetwork(colonyNetworkAddress).getReputationRootHash(),
+      amount,
+      amount,
+      _token,
+      block.timestamp
+    );
+
+    return globalRewardPayoutCount;
+  }
+
+  function claimRewardPayout(uint8 _payoutId, int256 _userReputation, int256 _totalReputation) public {
+    require(block.timestamp - rewardPayoutCycles[_payoutId].blockTimestamp < 5184000000); // 60 days
```

Comment:
`block.timestamp` is in seconds, not milliseconds. So this is actually 60000 days! While technically you won't need to change the tests (their very large numbers will still take us past the reward payout window once this is fixed), I'd prefer it if they were changed too.

Reply:
Sorry, i should have checked the documentation.